helios-testing/src/main/java/com/spotify/helios/testing/HeliosSoloLogService.java

```diff
@@ -137,4 +129,35 @@ protected Scheduler scheduler() {
     return Scheduler.newFixedDelaySchedule(0, 100, TimeUnit.MILLISECONDS);
   }
 
+  private class LogFollowJob implements Callable<Void> {
```

Comment:
So that the future can be failed with a checked exception instead of just a `RuntimeException`